runtime/lua/vim/loader.lua

```diff
@@ -253,7 +253,7 @@ function Loader.load(modpath, opts)
   local entry = Loader.read(modpath)
   if entry and Loader.eq(entry.hash, hash) then
     -- found in cache and up to date
-    chunk, err = load(entry.chunk --[[@as string]], '@' .. modpath, opts.mode, opts.env)
+    chunk, err = load(entry.chunk, --[[@as string]] '@' .. modpath, opts.mode, opts.env)
```

Comment:
there were two unrelated changes on `stylua runtime`. Should I leave them here?

Reply:
I added two empty lines, so that `-` would parse as bullet list in vimdoc

Reply:
They don't get added when I do this locally, so best remove them.